README.md

```diff
@@ -19,11 +19,22 @@ If you need help using or installing Node.js, please use the
 
 ## Resources for Newcomers
 
+### Official Resources
+
 * [Website](https://nodejs.org/en/)
 * [Contributing to the project](./CONTRIBUTING.md)
-* IRC (general questions): [#node.js on chat.freenode.net](https://webchat.freenode.net?channels=node.js&uio=d4)
 * IRC (node core development): [#node-dev on chat.freenode.net](https://webchat.freenode.net?channels=node-dev&uio=d4)
 
+### Unofficial Resources
+
+* IRC (general questions): [#node.js on chat.freenode.net](https://webchat.freenode.net?channels=node.js&uio=d4). Please
+see http://nodeirc.info/ for more information regarding the `#node.js` IRC channel.
+
+*Please note that unofficial resources are neither managed by -- nor necessarily
```

Comment:
What do two minus signs mean? Is that commonly used or is there an UTF char for that?